IFComp/t/entry_processing.t

```diff
@@ -0,0 +1,120 @@
+use strict;
+use warnings;
+use Test::More;
+
+use FindBin;
+use lib ("$FindBin::Bin/lib");
+use IFCompTest;
+my $schema = IFCompTest->init_schema();
+
+my $entry_directory =
+    Path::Class::Dir->new("$FindBin::Bin/platform_test_entries");
+
+$schema->entry_directory($entry_directory);
+
```

Comment:
Is there a compelling reason that this (setting the entry directory on the schema) and the corresponding Entry population below isn't part of IFCompTest?